source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+
+While you could download the code from Github and build it yourself, getting a copy can be as simple as running:
+
+.. code-block:: bash
+
+	sudo apt-get update
+	sudo apt install unbound -y
+
+This gives you a full, compiled, and running version of Unbound which behaves as a caching recursive DNS resolver out of the box for the local machine. 
+
+.. after it has been written, link to the local-stub to show how to compile and build.
+
+Do note that the current setup is only reachable on this machine.
+
+Testing the server locally
 --------------------------
 
-Using Unbound as the resolver for a home network.
+To verify that the server works correctly it’s a good idea to test it before committing the entire network to it. Luckily we can test this on the machine that you installed Unbound on (locally) and from any other machine (remotely) that will be using the resolver after we expose Unbound to the network.
+
+The command for local testing is:
+
+.. code-block:: bash
+
+	dig example.com @127.0.0.1
+
+Here we tell the :command:`dig` tool to look up the IP address for example.com, and to ask this information to the server running at the IP address ``127.0.0.1``, which is where our Unbound machine is running by default.
+We can verify that Unbound has indeed answered our query instead of the default resolver that is present on Ubuntu by default. In the output of every :command:`dig` command there is ``ANSWER SECTION`` which specifies the server which has answer the query under ``SERVER`` entry. The entry should be:
+
+.. code-block:: bash
+
+	;; SERVER: 127.0.0.1#53(127.0.0.1)
+
+In the next section we will be disabling the default Ubuntu resolver. To verify that we do it correctly there it is useful to know the address of the default resolver as a baseline. For this baseline, we also use a :command:`dig` query but time without specifying an IP address which the uses the machines default DNS resolver.
+
+.. code-block:: bash
+
+	dig example.com
+
+While the response should be the same, the ``SERVER`` in the response should look like:
+
+.. code-block:: bash
+
+	;; SERVER: 127.0.0.53#53(127.0.0.53)
+
+Note that the final IPv4 digit is 53 and not 1, as with our Unbound instance.
+
+Setting up for a single machine
+-------------------------------
+
+Now that we have configured and tested our Unbound server, we can tell our machine to use it by default. The nameserver (i.e. resolver) your machine uses by default is defined in :file:`/etc/resolv.conf`.
+While just changing this file will work as long as the machine doesn't reboot, the more permanent and better solution is to replace the file with our own. The reason for this is that the :file:`resolv.conf` file is a `symbolic link`. We will remove the link and create a new file ourselves.
```

Comment:
That the file is a symbolic link cannot be the reason that the changes do not persist. There is something not quite right here I think.